packages/keyring-controller/src/KeyringController.ts

```diff
@@ -2459,8 +2428,10 @@ export class KeyringController extends BaseController<
     }
 
     const keyring = keyringBuilder();
-    // @ts-expect-error Enforce data type after updating clients
-    await keyring.deserialize(data);
+    if (data) {
+      // @ts-expect-error Enforce data type after updating clients
+      await keyring.deserialize(data);
+    }
```

Comment:
Just want to make sure that this gets noticed: `KeyringController` now doesn't call `keyring.deserialize` if there's no data to pass

Reply:
I did saw it, and I think that actually make sense that way too. :)